### PR TITLE
shellcheck fix

### PR DESCRIPTION
**Pull request by lengyijun**

### CODE REVIEW

**Review by lehors — COMMENTED** [review]
Thank you for your contribution but for your PR to be acceptable it must have a sign-off (you can add one with git commit --amend -s).

File: fabcar/startFabric.sh [hunk 1]
```diff
@@ -11,9 +11,9 @@ set -e
 export MSYS_NO_PATHCONV=1
 starttime=$(date +%s)
 CC_SRC_LANGUAGE=${1:-"go"}
-CC_SRC_LANGUAGE=`echo "$CC_SRC_LANGUAGE" | tr [:upper:] [:lower:]`
+CC_SRC_LANGUAGE=$(echo "$CC_SRC_LANGUAGE" | tr '[:upper:]' '[:lower:]')
 
-if [ "$CC_SRC_LANGUAGE" = "go" -o "$CC_SRC_LANGUAGE" = "golang" ] ; then
+if [ "$CC_SRC_LANGUAGE" = "go" ] -o [ "$CC_SRC_LANGUAGE" = "golang" ] ; then
```
Comment (lehors) [inline fabcar/startFabric.sh:16]: This syntax is actually incorrect and fails:
$ ./startFabric.sh 
./startFabric.sh: line 16: [: too many arguments
The following will work:
```suggestion
if [ "$CC_SRC_LANGUAGE" = "go" ] || [ "$CC_SRC_LANGUAGE" = "golang" ] ; then
```

**Review by lehors — CHANGES_REQUESTED** [review]
No offense but I honestly don't see much value in this PR. Most of the changes are irrelevant, and one change simply breaks the script.

**Comment by nikhil550** [comment]
This PR is quite stale so i am going to close.

**Pull request closed without merging**